docs/user-guide/petset.md

```diff
@@ -0,0 +1,370 @@
+---
+---
+
+* TOC
+{:toc}
+
+__Terminology__
+
+Throughout this doc you will see a few terms that are sometimes used interchangeably elsewhere, that might cause confusion. This section attempts to clarify them.
```

Comment:
This is a user guide, added language to be more specific. im moving walk through sections to walkthrough.